catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet

```diff
@@ -0,0 +1,3433 @@
+          },
+          {
+            value: 15,
+            color: '98a39b',
+            description: 'cloud (land)',
+          },
+          {
+            value: 16,
+            color: '9899a3',
+            description: 'cloud (water)',
+          },
+          {
+            value: 17,
+            color: '135e9c',
+            description: 'water',
+          },          
+          {
+            value: 18,
+            color: 'e1e8ed',
+            description: 'cirrus cloud',
+          },
+          {
+            value: 19,
+            color: '9ea7ad',
+            description: 'cirrus cloud (thick)',
+          },
+          {
+            value: 20,
+            color: 'dffa11',
+            description: 'bright',
+          },
+          {
+            value: 21,
+            color: '383734',
+            description: 'topographic shadow',
+          },
+          {
+            value: 22,
+            color: '4f4f4e',
+            description: 'cloud (building) shadow',
+          },
+      ]
+     },
+     {
+      name: 'B432',
+      'gee:units': units.degree,
+      description: 'Illumination Factor used as input to ATCOR'
+     },
+     {
+      name: 'B433',
+      'gee:units': units.meter,
+      description: 'Path length between sensor and surface'
+     },
+     {
+      name: 'B434',
+      'gee:units': units.percent,
+      description: 'Sky View Factor used as input to ATCOR'
+     },
+     {
+      name: 'B435',
+      'gee:units': units.degree,
+      description: 'Slope used as input to ATCOR'
+     },
+     {
+      name: 'B436',
+      'gee:units': units.meter,
+      description: 'Smooth Surface Elevation used as input to ATCOR'
+     },
+     {
+      name: 'B437',
+      'gee:units': units.kilometer,
+      description: 'Visibility Index Map - sea level values of visibility index / total optical thickeness'
+     },
+     {
+      name: 'B438',
+      'gee:units': units.centimeter,
+      description: 'Water Vapor Column - (cm)*1000 ground-to-space used in ATCOR'
+     },
+     {
+      name: 'B439',
+      'gee:units': units.degree,
+      description: 'to-sensor Azimuth Angle'
+     },
+     {
+      name: 'B440',
+      'gee:units': units.degree,
+      description: 'to-sensor Zenith Angle'
+     },
+     {
+      name: 'B441',
+        'gee:classes': [
+            {
+            value: 0,
+            color: '239e2f',
+            description: 'mostly clear: <10% cloud cover',
+            },
+            {
+            value: 1,
+            color: 'f5ee1d',
+            description: 'partly cloudy: 10-50% cloud cover',
+            },
+            {
+            value: 2,
+            color: 'c40a0a',
+            description: 'mostly cloudy: >50% cloud cover',
+            },
+      ],
+      description: 'Weather Quality Indicator - estimated percentage of overhead cloud cover during acquisition'
+     },
+     {
+      name: 'B442',
+//      'gee:units': 'date yyyy-mm-dd',
```

Comment:
No unit needed for this band

Reply:
resolved in https://github.com/google/earthengine-catalog/pull/1058/commits/32addbee9bc71da76b33d5aac3f8a198263e491b